faker/providers/address/ro_RO/__init__.py

```diff
@@ -0,0 +1,130 @@
+        'Ocna Mures', 'Urlati', 'Strehaia', 'Tasnad', 'Cajvana', 'Tuzla',
+        'Sadova', 'Vlahita', 'Stei', 'Diosig', 'Cobadin', 'Gilau', 'Vladimirescu',
+        'Dancu', 'Bumbesti-Jiu', 'Busteni', 'Peretu', 'Cudalbi', 'Bosanci',
+        'Balotesti', 'Lunca Cetatuii', 'Dragalina', 'Fieni', 'Chisineu-Cris',
+        'Balan', 'Sandominic', 'Strejnicu', 'Baciu', 'Fundulea', 'Remetea',
+        'Fagetel (Remetea)', 'Ianca', 'Roseti', 'Breaza de Sus', 'Cornetu',
+        'Insuratei', 'Apahida', 'Berceni', 'Vicovu de Jos', 'Savinesti (Poiana Teiului)',
+        'Savinesti', 'Teius', 'Barbulesti', 'Plosca', 'Toflea', 'Magurele',
+        'Feldru', 'Anina', 'Negresti', 'Valea Mare (Negresti)', 'Peris',
+        'Fundeni', 'Giroc', 'Baile Borsa', 'Oituz', 'Rucar', 'Curcani',
+        'Babeni', 'Valea Mare (Babeni)', 'Rodna', 'Deta', 'Ruscova',
+        'Intorsura Buzaului', 'Pancota', 'Glina', 'Talmaciu', 'Copsa Mica',
+        'Motatei', 'Gugesti', 'Schela Cladovei', 'Sancraiu de Mures', 'Iernut',
+        'Targu Lapus', 'Maieru', 'Prejmer', 'Pogoanele', 'Dobroesti', 'Baraolt',
+        'Arbore', 'Homocea', 'Corund', 'Tufesti', 'Giarmata', 'Baia',
+        'Dumbraveni', 'Eforie Nord', 'Horodnic de Sus', 'Greci', 'Tudora', 'Straja',
+        'Rasinari', 'Sebis', 'Raducaneni', 'Siria', 'Paunesti', 'Saveni', 'Tunari',
+    )
+
+    states = (
+        ('AB', 'Alba'), ('AG', 'Argeș'), ('AR', 'Arad'), ('B', 'București'),
+        ('BC', 'Bacău'), ('BH', 'Bihor'), ('BN', 'Bistrița-Năsăud'),
+        ('BR', 'Brăila'), ('BT', 'Botoșani'), ('BV', 'Brașov'), ('BZ', 'Buzău'),
+        ('CJ', 'Cluj'), ('CL', 'Călărași'), ('CS', 'Caraș Severin'), ('CT', 'Constanța'),
+        ('CV', 'Covasna'), ('DB', 'Dâmbovița'), ('DJ', 'Dolj'), ('GJ', 'Gorj'),
+        ('GL', 'Galați'), ('GR', 'Giurgiu'), ('HD', 'Hunedoara'), ('HR', 'Harghita'),
+        ('IF', 'Ilfov'), ('IL', 'Ialomița'), ('IS', 'Iași'), ('MH', 'Mehedinți'),
+        ('MM', 'Maramureș'), ('MS', 'Mureș'), ('NT', 'Neamț'), ('OT', 'Olt'),
+        ('PH', 'Prahova'), ('SB', 'Sibiu'), ('SJ', 'Sălaj'), ('SM', 'Satu Mare'),
+        ('SV', 'Suceava'), ('TL', 'Tulcea'), ('TM', 'Timiș'), ('TR', 'Teleorman'),
+        ('VL', 'Vâlcea'), ('VN', 'Vrancea'), ('VS', 'Vaslui'),
+    )
+
+    def street_prefix(self):
+        """
+        :example 'Strada'
+        """
+        return self.random_element(self.street_prefixes)
+
+    def secondary_address(self):
+        """
+        :example 'Bl. 123 Sc. 2 Ap. 15'
+        """
+        return self.numerify(
+            self.random_element(
+                self.secondary_address_formats))
+
+    def city_name(self):
+        return self.random_element(self.cities)
+
+    def city_with_postcode(self):
+        return self.postcode() + " " + self.random_element(self.cities)
+
+    def state(self):
+        """
+        example: u'Timiș'
+        """
+        return self.random_element(self.states)[1]
```

Comment:
can you rename this to `administrative_unit` and alias it to `state`?

```suggestion
    def administrative_unit(self):
        """
        example: u'Timiș'
        """
        return self.random_element(self.states)[1]
    
    state = administrative_unit
```

Reply:
Updated, thanks.